stock_getter/backend_stockgetter/stock_service/tests.py

```diff
@@ -1,64 +1,34 @@
-from django.test import TestCase
-from django.urls import reverse, resolve
-from django.test import Client
-from .views import StockListServiceAPIView, StockDataServiceAPIView
 import requests
+from django.test import Client, TestCase
+from django.urls import resolve, reverse
+from parameterized import parameterized
```

Comment:
Any chance we could add a comment on how parameterized works. I looked it up and understand after looking it up but it would be nice to be in the code for anyone else that's not familiar.

Reply:
I can add some description here, but one of the concerns I have is that this package could be in the future used in a lot of different files and it would not make sense to put description in each of them. Also, we could make a case that for a lot of other packages we ultimately need to look up documentation of how it works.

Can we maybe put explanation in the backend readme file? I can add this in part for unit tests or if you have some other suggestion about a better place for adding it.

Reply:
Description can be added, but one of the concerns I have is that this package could be used in a lot of different files in the future and it would not make sense to put description in each of them. Also, we could make a case that for a lot of other packages we are using that ultimately need to look up documentation of how they work. If you agree we can check in this, otherwise I can explain somewhere else for how to use parameterized.